migration/migration.go

```diff
@@ -338,6 +341,9 @@ func PopulateCommonTypes(ctx context.Context, db *gorm.DB, witr *workitem.GormWo
 	if err := createOrUpdatePlannerItemExtension(workitem.SystemFeature, ctx, witr, db); err != nil {
 		return errs.WithStack(err)
 	}
+	if err := createOrUpdatePlannerItemExtension(workitem.SystemEpic, ctx, witr, db); err != nil {
```

Comment:
> The story talks about adding features in the core that would support the use case that an Epic represents, not just add an Epic.

I don't quite get what you're saying @aslakknutsen . This PR adds an epic and a link type for an epic. Shouldn't that be enough to fulfill these points from the functional acceptance criteria of story https://github.com/fabric8io/fabric8-planner/issues/657?

> * The user can navigate to the work item create dialog (see #501) and create a new work item of type "Epic".
> * The dialog behaves the same as the normal create work item dialog.